### PR TITLE
added JDBC DAOs implementations for bucket, user, order

### PR DESCRIPTION
**Pull request by tsehelnyk**
Not finished. Custom Exception not implemented yet.
Check my new DAOs, please. =)

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/jdbc/BucketDaoJdbcImpl.java [hunk 1]
```diff
@@ -1,38 +1,134 @@
...
+            throw new RuntimeException();
+        }
+        return bucket;
     }
 
     @Override
     public boolean delete(Long id) {
-        return false;
+        String query = String.format("DELETE FROM %s WHERE bucket_id = ?;", BUCKETS_TABLE_NAME);
+
```
[the remark this answers is not in the document]
Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/dao/jdbc/BucketDaoJdbcImpl.java:111]: it is not necessary cascade deleting is configured very well)
i've checked)

File: src/main/java/mate/academy/internetshop/controller/RegistrationController.java [hunk 2]
```diff
@@ -34,7 +34,9 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             newUser.setName(req.getParameter("name"));
             newUser.setLogin(req.getParameter("login"));
             newUser.setPassword(req.getParameter("psw"));
-            newUser.addRole(new Role(Role.RoleName.USER));
+            Role role = new Role(2L);
```
[the remark this answers is not in the document]
Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/controller/RegistrationController.java:41]: i'm in process of replacing Exceptions)

**Review by Sofasmile — COMMENTED** [review]
good) I'm waiting for exception)

re: how do you like mine getAllUsers?))

File: src/main/java/mate/academy/internetshop/dao/jdbc/BucketDaoJdbcImpl.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/dao/jdbc/BucketDaoJdbcImpl.java]: > good) I'm waiting for exception)

how do you like mine getAllUsers?))

File: src/main/java/mate/academy/internetshop/Main.java [hunk 3]
```diff
@@ -1,19 +1,8 @@
...
-import mate.academy.internetshop.model.Bucket;
-import mate.academy.internetshop.model.Item;
-import mate.academy.internetshop.model.Order;
-import mate.academy.internetshop.model.User;
-import mate.academy.internetshop.service.BucketService;
-import mate.academy.internetshop.service.ItemService;
-import mate.academy.internetshop.service.OrderService;
-import mate.academy.internetshop.service.UserService;
-
 public class Main {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/Main.java]: давай нарешті видалимо цей клас

File: src/main/java/mate/academy/internetshop/controller/RegistrationController.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/controller/RegistrationController.java:41]: done

File: src/main/java/mate/academy/internetshop/Main.java (hunk 3 above)
Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/Main.java]: )

Comment (tsehelnyk) [reply src/main/java/mate/academy/internetshop/Main.java]: done!)

**Pull request merged**